src/Simplex/Chat.hs

```diff
@@ -759,6 +763,12 @@ processChatCommand = \case
             quoteData ChatItem {chatDir = CIGroupSnd, content = CISndMsgContent qmc} membership' = pure (qmc, CIQGroupSnd, True, membership')
             quoteData ChatItem {chatDir = CIGroupRcv m, content = CIRcvMsgContent qmc} _ = pure (qmc, CIQGroupRcv $ Just m, False, m)
             quoteData _ _ = throwChatError CEInvalidQuote
+    CTLocal -> do
+      nf@NoteFolder {noteFolderId} <- withStore $ \db -> getNoteFolder db user chatId
+      -- TODO: files, voice, etc.
+      msg <- createSndMessage (XMsgNew . MCSimple $ extMsgContent mc Nothing) (NoteFolderId noteFolderId)
+      ci <- saveSndChatItem user (CDLocalSnd nf) msg (CISndMsgContent mc)
```

Comment:
It's not necessary to create records in `messages` table, only chat items. messages are only used to store "service messages", which are linked via msg_deliveries to messages as sent to / received from agent connections. Since we don't need any connections here, it's ok to simply put msg content (mc) into chat items, w/t even wrapping them into protocol events and encoding.

Reply:
it may cause problems though if we ever allow to forward them to conversations with connections, because there they'd have to be wrapped. Perhaps in notes we should check for lower limit than protocol message limit (it wasn't present anywhere in chat code explicitly until #3519, in that branch it's maxChatMsgSize).